constants/chainparams.go

```diff
@@ -0,0 +1,8 @@
+package constants
```

Comment:
Since Liquid runs on Bitcoin, it has the same `HDPrivateKeyID` and `HDPublicKeyID`, see https://github.com/btcsuite/btcd/blob/v0.23.4/chaincfg/params.go#L405-L407. In my opinion we can delete this file and directly use:
```
var chainParams = &chaincfg.MainNetParams
```

Reply:
The difference between Liquid and Bitcoin is `HDCoinType`.